Formula/lm4tools.rb

```diff
@@ -0,0 +1,23 @@
+class Lm4tools < Formula
+  desc "Tools for TI Stellaris Launchpad boards"
+  homepage "https://github.com/utzig/lm4tools"
+  url "https://github.com/utzig/lm4tools/archive/v0.1.2.tar.gz"
+  sha256 "2619ae4350a3a2190a12f6fae03c86a55975b85f47b891b694f0a7a17f7369a6"
+
+  depends_on "pkg-config" => :build
+  depends_on "libusb"
+
+  def install
+    cd "lm4flash"
+    system "make", "all"
+    bin.install "lm4flash"
+
+    cd "../lmicdiusb"
+    system "make", "all"
+    bin.install "lmicdi"
+  end
+
+  test do
+    system "#{bin}/lm4flash", "-V"
```

Comment:
We are going to need a test that does more than that…

Reply:
The software needs connected hardware to do anything useful. I can write a proper test, but that would break CI.

Reply:
Could also grep for the version, or even try flashing *any* file which should output "Unable to find any ICDI devices" when no device is connected (like in a CI). Of course, this would fail if run locally with an actual device connected...

Reply:
> try flashing any file which should output "Unable to find any ICDI devices" when no device is connected

This is good. Or flashing something that is an invalid file and check that it says so (if it does).

Reply:
I implemented the suggested test. It will of course fail if a device is connected, but that is probably obvious if anyone runs the test and sees that "Unable to find any ICDI devices" didn't match the stderr.